clients/pkg/promtail/promtail_test.go

```diff
@@ -169,23 +169,23 @@ func TestPromtail(t *testing.T) {
 		expectedEntries[entry] = i
 	}
 	lbls := []labels.Labels{}
-	lbls = append(lbls, labels.Labels{
-		labels.Label{Name: "action", Value: "GET"},
-		labels.Label{Name: "filename", Value: dirName + "/logs/testPipeline.log"},
-		labels.Label{Name: "job", Value: "varlogs"},
-		labels.Label{Name: "localhost", Value: ""},
```

Comment:
Note that label with empty value cannot be created when using labels builder in Prometheus (via some paths anyway), so I've changed this test to use `address="localhost"` label instead.